api/src/main/java/org/openmrs/module/patientqueueing/api/dao/PatientQueueingDao.java

```diff
@@ -11,21 +11,84 @@
 
 import org.apache.commons.logging.Log;
```

Comment:
Did you take a look at our recommendation for logging? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging